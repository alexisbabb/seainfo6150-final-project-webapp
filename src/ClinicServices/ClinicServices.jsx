import React from 'react'; 
import styles from "./ClinicServices.module.css";

const ClinicServices = (props) => {
    return (
        <article className = {styles.pageContent}>
            <header className = {styles.pageTitle} >
            Get vaccinations, microchipping, and dog licenses!

            </header>
            <div className = {styles.pageDetails}>
                <img className = {styles.pageImage} src="https://www.mercypetclinic.org/wp-content/uploads/2017/12/MPC_Dog_Vaccine_Main.jpg" alt="Clinic Services"/>
                <div>
                    <p><u>Location:</u> DHA Wilmington (701 A St Wilmington DE 19801)</p>
                    <br></br>
                    <br></br>
                    <u>Time:</u> Clinic events are held the first Saturday every month from 10am-12pm Eastern
                    <br></br>
                    <br></br>
                    <u>Services and Pricing:</u>
                    <br></br>
                    <ul>
                        <li>Rabies Vaccination - $20 each</li>
                        <br></br>
                        <li>FVR-CP (Feline Distemper) Vaccination - $20 each</li>
                        <br></br>
                        <li>DHP-PV (Canine Distemper) Vaccination - $20 each</li>
                        <br></br>
                        <li>24PetWatch Microchip - $20</li>
                        <br></br>
                        <li>Rabies Vaccination - $20</li>
                        <br></br>
                        <li>Rabies Vaccination - $20</li>
                        <br></br>
                    </ul>
                </div>
            </div>
        </article>
    )
}



export default ClinicServices