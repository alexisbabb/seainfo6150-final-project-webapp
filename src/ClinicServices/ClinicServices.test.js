import React from "react";
import ClinicServices from "./ClinicServices"

describe("ClinicServices tests", () => {
    it("renders correctly", () => {
        const src = "https://www.mercypetclinic.org/wp-content/uploads/2017/12/MPC_Dog_Vaccine_Main.jpg";
        const { container } = render(<ClinicServices src={src}/>);
        expect(container).toMatchSnapshot();
    });
});