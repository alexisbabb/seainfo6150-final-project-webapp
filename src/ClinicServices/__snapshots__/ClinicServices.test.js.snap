// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ClinicServices tests renders correctly 1`] = `
<div>
  <article
    class="pageContent"
  >
    <header
      class="pageTitle"
    >
      Get vaccinations, microchipping, and dog licenses!
    </header>
    <div
      class="pageDetails"
    >
      <img
        alt="Clinic Services"
        class="pageImage"
        src="https://www.mercypetclinic.org/wp-content/uploads/2017/12/MPC_Dog_Vaccine_Main.jpg"
      />
      <div>
        <p>
          <u>
            Location:
          </u>
           DHA Wilmington (701 A St Wilmington DE 19801)
        </p>
        <br />
        <br />
        <u>
          Time:
        </u>
         Clinic events are held the first Saturday every month from 10am-12pm Eastern
        <br />
        <br />
        <u>
          Services and Pricing:
        </u>
        <br />
        <ul>
          <li>
            Rabies Vaccination - $20 each
          </li>
          <br />
          <li>
            FVR-CP (Feline Distemper) Vaccination - $20 each
          </li>
          <br />
          <li>
            DHP-PV (Canine Distemper) Vaccination - $20 each
          </li>
          <br />
          <li>
            24PetWatch Microchip - $20
          </li>
          <br />
          <li>
            Rabies Vaccination - $20
          </li>
          <br />
          <li>
            Rabies Vaccination - $20
          </li>
          <br />
        </ul>
      </div>
    </div>
  </article>
</div>
`;
